### PR TITLE
feat: Add configuration option to delay message queue resume

### PR DESCRIPTION
**Pull request by valentin-dimov**
Companion to https://github.com/EVerest/libocpp/pull/283

### CODE REVIEW

File: modules/OCPP201/manifest.yaml [hunk 1]
```diff
@@ -24,6 +24,10 @@ config:
     description: If true websocket can be disconnected and connected externally
     type: boolean
     default: false
+  MessageQueueResumeDelay:
```
Comment (hikinggrass) [inline modules/OCPP201/manifest.yaml:27]: Probably makes sense to have this in the OCPP (1.6) module as well

Comment (valentin-dimov) [reply modules/OCPP201/manifest.yaml:27]: We would need to extend the libocpp PR a bit as well, in order to allow passing that setting to the 1.6 chargepoint. Should we do that as part of this PR?

Comment (valentin-dimov) [reply modules/OCPP201/manifest.yaml:27]: Done

**Review by hikinggrass — APPROVED** [review]
please update libocpp rev then this should build and be able to merge

**Pull request merged**